egg/zoo/compo_vs_generalization/train.py

```diff
@@ -85,6 +85,12 @@ def get_params(params):
         default=0.99999,
         help="Early stopping threshold on accuracy (defautl: 0.99999)",
     )
+    parser.add_argument(
+        "--fixed_length",
+        type=str,
+        default="True",
+        help="Whether the messages are variable or fixed length",
+    )
```

Comment:
You can make this a boolean flag with `action="store_true"`

```python
    parser.add_argument(
        "--fixed_length",
        action="store_true",
        help="Whether the messages are variable or fixed length",
    )
```

The argument will automatically be converted to a boolean as well